### ISSUE TITLE
test with HTTP::Headers 6.00 installed

### ISSUE THREAD

**Issue by mw333** [PerlDancer/Dancer#710]
While testing Dancer [1.3080](via command line cpan) I noticed 
t/12_response/04_charset_server.t .............. skipped: HTTP::Headers 5.827 required (use of content_type_charset)
and so I installed HTTP::Headers ..

> install HTTP::Headers
> Running install for module 'HTTP::Headers'
> Running make for G/GA/GAAS/HTTP-Message-6.02.tar.gz

and that tested and installed fine:

PERL_DL_NONLAZY=1 /usr/bin/perl "-MExtUtils::Command::MM" "-e" "test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/common-req.t .......... ok  
t/headers-auth.t ........ ok  
t/headers-etag.t ........ ok  
t/headers-util.t ........ ok  
t/headers.t ............. ok  
t/http-config.t ......... ok  
t/message-charset.t ..... ok  
t/message-decode-xml.t .. ok  
t/message-old.t ......... ok  
t/message-parts.t ....... ok  
t/message.t ............. ok  
t/request.t ............. ok  
t/response.t ............ ok  
t/status-old.t .......... ok  
t/status.t .............. ok  
All tests successful.
Files=15, Tests=529,  1 wallclock secs ( 0.09 usr  0.03 sys +  0.51 cusr  0.08 csys =  0.71 CPU)
Result: PASS
  GAAS/HTTP-Message-6.02.tar.gz
  /usr/bin/make test -- OK
Running make install
...
but now a fresh
clean Dancer; test Dancer has a few extra problems.

basically these are generally:
500 Can't locate object method "_is_html" via package "HTTP::Headers"

-- This causes installation to fail (without a force)

.. This could all be just my bad luck, 
.. A problem with the latest HTTP::Headers (never used the module myself)
.. Or that the machine has perl 5.10 on it.
.. Or these tests never passed.

More error reports below:
